### ISSUE TITLE
UnusedPrivateElementsSniff should ignore variables as references

### ISSUE THREAD

**Issue by ostrolucky** [slevomat/coding-standard#668]
`SlevomatCodingStandard\Sniffs\Classes\UnusedPrivateElementsSniff.WriteOnlyProperty` should not be reported on following

```php
class foo {
  private $bar;

  public function __construct(&$bar) {
    $this->bar = &$bar;
  }

  public function bar() {
    $this->bar = 0;
  }
}
```

Upon write to this internal property, external variable is being changed.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/3a590a405219ef184313163dd5f52036ee99bc3b
